hpcpy/utilities.py

```diff
@@ -33,7 +34,7 @@ def shell(
     subprocess.CalledProcessError
     """
```

Comment:
`shell` is no longer being used, was this intended? If so it should be removed from the call signature and docstring.

Reply:
Yes, it is no longer needed. Will remove.

Reply:
Would be a good idea to say why this change was made in case someone does some code sleuthing later and is thinking of changing it back again.

This is hard-earned wisdom. Often from sleuthing my own damn code changes and trying to figure out why I changed stuff.

Reply:
![image](https://github.com/user-attachments/assets/1d67d4c2-b351-466b-a3b3-d97a24956a11)

I can answer this in 3 words. Security, security, security. Shlex splitting is safer than a straight shell call.